source/broadband/webpa_notification.c

```diff
@@ -1369,6 +1495,35 @@ static WDMP_STATUS processParamNotification(ParamNotify *paramNotify,
 	return status;
 }
 
+static WDMP_STATUS processParamNotificationRetry(unsigned int *cmc, char **cid)
+{
+	char *dbCID = NULL, *dbCMC = NULL;
+	WDMP_STATUS status = WDMP_FAILURE;
+
+	WalInfo("processParamNotificationRetry........\n");
+
+	dbCMC = getParameterValue(PARAM_CMC);
+	if (NULL != dbCMC) 
+	{
+	        dbCID = getParameterValue(PARAM_CID);
+	        if (NULL == dbCID) 
+		{
+			WAL_FREE(dbCMC);
+			WalError("Error dbCID is NULL!\n");
+			return status;
+		}
+	} 
+	else 
+	{
+	        WalError("Error dbCMC is NULL!\n");
+	        return status;
+	}
+	(*cmc) = atoi(dbCMC);
+	(*cid) = dbCID;
+	WAL_FREE(dbCMC);
+	return WDMP_SUCCESS;	
```

Comment:
At line 1513, status returned, here WDMP_SUCCESS . if variable is defined , use it else remove the variable and return direct macro, make it consistent at both the places